src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,136 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "Nikitos";
+    private static final int DEFAULT_AGE = 26;
+    private static final String DEFAULT_PASSWORD = "qwertyu1213";
+    private static final int MAX = 100;
+    private static final int MIN = 18;
+    private static final String EMPTY = "";
+    private static RegistrationServiceImpl registrationService;
+    private static StorageDao storageDao;
```

Comment:
Don't use StorageDao field, you can get storage directly through Storage.people.